index.html

```diff
@@ -39,6 +39,80 @@ <h1>I'm José</br>
             </li>
         </ul>
     </section>
+    <section class="works">
+        <article>
+            <img src="images/Snapshoot Portfolio.svg">
+            <div>
+                <h2>Tonic</h2>
+                <div class="ref">
+                    <p class="category">Canopy</p>
+                    <p class="rest">Back End Dev</p>
+                    <p class="rest">2015</p>
+                </div>
+                <p class="copy">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="r1">
+                    <button class="a1">html</button>
+                    <button class="a1">css</button>
+                    <button class="a1">javascript</button>
+                </div>
+                <div class="action">
+                    <button class="a2">See project</button>
```

Comment:
Great job team 🤝

- [ ] Kindly make sure the button text is well centered on the button. This will make your design a replica of the design provided in the Figma template.
![your button](https://user-images.githubusercontent.com/63917725/192119641-36cfb7ad-6889-430d-8180-fd4c96dd16da.PNG)